### PR TITLE
Add p-label

### PR DESCRIPTION
**Pull request by lyubomir-popov**
## Done

Fixes https://github.com/canonical-web-and-design/vanilla-framework/issues/3201

## QA

- Open docs/patterns/labels
- Review the The "Default" section at the top

### Check if PR is ready for release

If this PR contains Vanilla SCSS code changes, it should contain the following changes to make sure it's ready for the release:

- [x] PR should have one of the following labels to automatically categorise it in release notes:
  - `Feature 🎁`, `Breaking Change 💣`, `Bug 🐛`, `Documentation 📝`, `Maintenance 🔨`.
- [x] Vanilla version in `package.json` should be updated relative to the [most recent release](https://github.com/canonical-web-and-design/vanilla-framework/releases/latest), following semver convention:
  - if CSS class names are not changed it can be bugfix relesase (x.x.**X**)
  - if CSS class names are changed/added/removed it should be minor version (x.**X**.0)
  - see the [wiki for more details](https://github.com/canonical-web-and-design/vanilla-framework/wiki/Release-process#pre-release-tasks)
- [x] Any changes to component class names (new patterns, variants, removed or added features) should be listed on the [component status page](https://github.com/canonical-web-and-design/vanilla-framework/blob/master/templates/docs/component-status.md).
- [x] Documentation side navigation should be updated with the relevant labels.

## Screenshots

image

### CODE REVIEW

**Comment by bartaz** [comment]
@lyubomir-popov These sections in the PR template are there to remind us about stuff to include in the PR: 😉 

Screenshot 2021-06-23 at 15 05 09

Could you please:
- [ ] add a label ("Feature") 
- [x] the version is already bumped up, so nothing to update there
- [ ] add new class name to components status page
- [ ] update docs side navigation for label to show it was updated

Adding a screenshot very often helpful as well, because it shows what to look for in the PR.

**Review by anthonydillon — APPROVED** [review]
Code and example LGTM thanks @lyubomir-popov. 

One thing, is there a worry that someone might override the disabled text colour and that would lead to unexpected behaviour. Like this label changing? Not a blocker as I think theme work here would solve this issue.

**Comment by lyubomir-popov** [comment]
@anthonydillon fixed. I removed the --disabled colour as it is not jused anywhere in our org. Actual disable states are achieved with opacity.

**Comment by lyubomir-popov** [comment]
Reasdy for another look

File: templates/docs/component-status.md [hunk 1]
```diff
@@ -24,6 +24,12 @@ When we add, make significant updates, or deprecate a component we update their
   <tbody>
     <!-- 2.32 -->
     <tr>
+      <th><a href="/docs/patterns/buttons#neutral">Labels / Default</a></th>
```
Comment (bartaz) [inline templates/docs/component-status.md:27]: Some copy/paste issues here:

```suggestion
      <th><a href="/docs/patterns/labels#default">Labels / Default</a></th>
```

File: templates/docs/component-status.md [hunk 2]
```diff
@@ -24,6 +24,12 @@ When we add, make significant updates, or deprecate a component we update their
...
+      <td><div class="p-label--new">Deprecated</div></td>
```
Comment (bartaz) [inline templates/docs/component-status.md:28]:
```suggestion
      <td><div class="p-label--new">New</div></td>
```

**Comment by lyubomir-popov** [comment]
Thanks for the suggestins @bartaz I've committed them.

**Review by bartaz — APPROVED** [review]
LGTM, thanks

**Pull request merged**